syntax = "proto3";

package course;

service CourseService {
  rpc GetCourseById (GetCourseByIdRequest) returns (GetCourseByIdResponse);
}

message GetCourseByIdRequest {
  int32 id = 1;
}

message GetCourseByIdResponse {
  int32 id = 1;
  string title = 2;
  string description = 3;
}

